README.md

```diff
@@ -17,52 +20,71 @@ Zeppelin is a moderation bot for Discord, designed with large servers and reliab
 - Starboard
 - And more!
 
-See https://zeppelin.gg/ for more details.
+See <https://zeppelin.gg/> for more details.
 
 ## Development
-These instructions are intended for bot development only.
 
-👉 **No support is offered for self-hosting the bot!** 👈
+These instructions are intended for bot development only, they are not recommended for self-hosting.
```

Comment:
I think the bolded "No support is offered for self-hosting" should be kept. That is still the case, and having it bolded makes it eye-catching.